### PR TITLE
Creates Aysha 'about me' page with bio and CSC roles details

### LINKED ISSUE #200: Aysha needs an 'about me' page

### PR DESCRIPTION
**Pull request by dangerdika**

### CODE REVIEW

File: _team_member/aysha.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: teammember
+team-member-name: Aysha Luis
+team-member-photo: /assets/img/team/Aysha.jpg
+job-role: Fellow in Clinical AI
+---
+
+### Bio
+Dr Aysha Luis is a senior neuroradiology fellow at Guy's and St Thomas' NHS Trust, specialising in both adult and paediatric neuroradiology. She completed subspecialty training on the Pan-London Neuroradiology Fellowship programme, rotating through several major neuroscience centres in London. Her main interests include neuro-oncology, advanced imaging and fetal MRI.
```
Comment (AnilMistry) [inline _team_member/aysha.md]: oxford comma after "advanced imaging,"

File: _team_member/aysha.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+### CSC Team Role
+Aysha joined the CSC in August 2022, and has been working on the deployment, evaluation and implementation of validated AI tools into the clinical workflow. She has a particular interest in establishing an efficient data curation process to facilitate clinical AI evaluations at scale.
```
Comment (AnilMistry) [inline _team_member/aysha.md]: another oxford comma after "evaluation,"

**Review by AnilMistry — CHANGES_REQUESTED** [review]
Works in website locally, just two oxford commas, then good to go.

**Comment by dangerdika** [comment]
I have consulted with @aysha1207 about the adding of the commas and she stated she has no preference so I have added the commas as suggested @AnilMistry.

**Review by AnilMistry — APPROVED** [review]

**Pull request merged**